readme.md

```diff
@@ -140,4 +143,4 @@ There are some configuration options exposed for you in the config file `showcas
 By default, all showcase tables are prefixed with `showcase_`. You may change this to whatever you desire.
 
 #### Middleware
-By default, all admin routes for Showcase are passed through `web` and `auth`. You may change this array in config to add or remove middleware as you want.
\ No newline at end of file
+By default, all admin routes for Showcase are passed through `web` and `auth`. You may change this array in config to add or remove middleware as you want.
```

Comment:
Diff is showing a change here, but I'm not seeing what...?

Reply:
hmmm, maybe the line ending?